tiled/server/settings.py

```diff
@@ -46,13 +108,41 @@ class Settings(BaseSettings):
     )
     database_init_if_not_exists: bool = False
     expose_raw_assets: bool = True
+    admins: list[Admin] = Field(default_factory=list)
+    access_control: Annotated[
+        Optional[AccessPolicy], "AccessPolicy to apply to all Trees",
+        BeforeValidator(partial(_optional_get, typ=AccessPolicy)),
+    ]
+
+    uvicorn: Uvicorn = Field(Uvicorn())
 
     model_config = SettingsConfigDict(
         env_prefix="TILED_",
         nested_model_default_partial_update=True,
         env_nested_delimiter="_",
     )
 
+    def check_scalable(self):
```

Comment:
todo: Make this a validator for the class- currently only distinguishes the case where secret_keys has been explicitly set as an empty list or database uri has been explicitly set to blank, we need to check that those keys have been explicitly set at all.